src/fmt.rs

```diff
@@ -443,13 +439,38 @@ pub fn print_install_verbose(config: &Config, actions: &Actions, devel: &HashSet
         });
 
         for pkg in &install {
-            println!(
```

Comment:
if println is going to overpad by the amount of characters we added by colouring. Can't we just do `package_len -= pkg_str.len() -  pkg.pkg.db().unwrap().name().len() - pkg.pkg.name().len() - 1` to calculate the difference like you have but keep using the `{:<package_len$}` and similar formatters instead of repeating strings.